packs/chef/actions/lib/shellhelpers.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import sys
+import os
+import subprocess
+import select
+import io
+
+# Starts command in subprocess processing output as it appears.
+def shell_out(command, env={}):
```

Comment:
Thanks Thomaz for explanaition, it's  an amazing feature :) I couldn't just imagine it that it works kinda static variables...